### ISSUE TITLE
F1 input is hard coded, caused user issues

### ISSUE THREAD

**Issue by mahoneyt944** [libretro/mame2003-plus-libretro#1287]
This being hardcoded has caused a user using F1 as an input button issues. Needs looked at.

https://github.com/libretro/mame2003-plus-libretro/blob/6b8ac2910ce768f8e19597db6c5c4f13e2f502bb/src/vidhrdw/m92_vidhrdw.c#L591-L608

**Comment by grant2258** [comment]
what was the issue caused ?

**Comment by mahoneyt944** [comment]
From my understanding, the users setup uses F1 and F2 as it's main action buttons and since F1 is hardcoded in the video update code above, it constantly toggles this unintentionally. I advised the user to set the input interface to retropad, which should fix the issue. But I figured I should report this here in case we wanted to alter it in some way for future cases. 

Given the unique instance of the user's inputs, I don't think this will be a big issue. But inputs that cannot be remapped aren't ideal. Maybe we should set these types of actions to a common remappable input.
